2024-10-24 12:09:15,999 WARNING Error fetching tasks: /home/frank/eazy_manage_backend/api/v1/routes/tasks.py 68
Traceback (most recent call last):
  File "/home/frank/eazy_manage_backend/api/v1/routes/tasks.py", line 57, in get_tasks
    serialized_data = [
  File "/home/frank/eazy_manage_backend/api/v1/routes/tasks.py", line 61, in <listcomp>
    'first_name': (lambda task : task[2].to_dict()).first_name,
AttributeError: 'function' object has no attribute 'first_name'
2024-10-24 12:09:21,286 WARNING Error fetching tasks: /home/frank/eazy_manage_backend/api/v1/routes/tasks.py 68
Traceback (most recent call last):
  File "/home/frank/eazy_manage_backend/api/v1/routes/tasks.py", line 57, in get_tasks
    serialized_data = [
  File "/home/frank/eazy_manage_backend/api/v1/routes/tasks.py", line 61, in <listcomp>
    'first_name': (lambda task : task[2].to_dict()).first_name,
AttributeError: 'function' object has no attribute 'first_name'
2024-10-24 12:14:47,179 INFO Logging setup successfully: /home/frank/eazy_manage_backend/./app.py 43
2024-10-24 12:14:47,180 INFO DataBase set up successfully: /home/frank/eazy_manage_backend/./app.py 55
2024-10-24 12:14:47,655 INFO database set up successfully: /home/frank/eazy_manage_backend/./app.py 60
2024-10-24 12:14:47,663 INFO Migrations set up successfully: /home/frank/eazy_manage_backend/./app.py 68
2024-10-24 12:14:47,676 INFO Blueprints registered successfully: /home/frank/eazy_manage_backend/./app.py 76
2024-10-24 12:14:47,676 INFO Mail service setup successfully: /home/frank/eazy_manage_backend/./app.py 89
2024-10-24 12:14:47,676 INFO App created successfully: /home/frank/eazy_manage_backend/./app.py 99
2024-10-24 12:14:49,215 WARNING Error fetching tasks: /home/frank/eazy_manage_backend/api/v1/routes/tasks.py 68
Traceback (most recent call last):
  File "/home/frank/eazy_manage_backend/api/v1/routes/tasks.py", line 57, in get_tasks
    serialized_data = [
  File "/home/frank/eazy_manage_backend/api/v1/routes/tasks.py", line 61, in <listcomp>
    'first_name': (task[2].to_dict()).first_name,
AttributeError: 'dict' object has no attribute 'first_name'
2024-10-24 12:14:52,022 WARNING Error fetching tasks: /home/frank/eazy_manage_backend/api/v1/routes/tasks.py 68
Traceback (most recent call last):
  File "/home/frank/eazy_manage_backend/api/v1/routes/tasks.py", line 57, in get_tasks
    serialized_data = [
  File "/home/frank/eazy_manage_backend/api/v1/routes/tasks.py", line 61, in <listcomp>
    'first_name': (task[2].to_dict()).first_name,
AttributeError: 'dict' object has no attribute 'first_name'
2024-10-24 12:15:58,760 INFO Logging setup successfully: /home/frank/eazy_manage_backend/./app.py 43
2024-10-24 12:15:58,760 INFO DataBase set up successfully: /home/frank/eazy_manage_backend/./app.py 55
2024-10-24 12:15:58,880 INFO database set up successfully: /home/frank/eazy_manage_backend/./app.py 60
2024-10-24 12:15:58,885 INFO Migrations set up successfully: /home/frank/eazy_manage_backend/./app.py 68
2024-10-24 12:15:58,897 INFO Blueprints registered successfully: /home/frank/eazy_manage_backend/./app.py 76
2024-10-24 12:15:58,897 INFO Mail service setup successfully: /home/frank/eazy_manage_backend/./app.py 89
2024-10-24 12:15:58,897 INFO App created successfully: /home/frank/eazy_manage_backend/./app.py 99
2024-10-24 12:16:43,219 INFO Logging setup successfully: /home/frank/eazy_manage_backend/./app.py 43
2024-10-24 12:16:43,220 INFO DataBase set up successfully: /home/frank/eazy_manage_backend/./app.py 55
2024-10-24 12:16:43,769 INFO database set up successfully: /home/frank/eazy_manage_backend/./app.py 60
2024-10-24 12:16:43,792 INFO Migrations set up successfully: /home/frank/eazy_manage_backend/./app.py 68
2024-10-24 12:16:43,815 INFO Blueprints registered successfully: /home/frank/eazy_manage_backend/./app.py 76
2024-10-24 12:16:43,815 INFO Mail service setup successfully: /home/frank/eazy_manage_backend/./app.py 89
2024-10-24 12:16:43,816 INFO App created successfully: /home/frank/eazy_manage_backend/./app.py 99
2024-10-24 12:16:45,224 WARNING Error fetching tasks: /home/frank/eazy_manage_backend/api/v1/routes/tasks.py 68
Traceback (most recent call last):
  File "/home/frank/eazy_manage_backend/api/v1/routes/tasks.py", line 57, in get_tasks
    serialized_data = [
  File "/home/frank/eazy_manage_backend/api/v1/routes/tasks.py", line 61, in <listcomp>
    'first_name': task[2].to_dict().first_name,
AttributeError: 'dict' object has no attribute 'first_name'
2024-10-24 12:16:46,858 WARNING Error fetching tasks: /home/frank/eazy_manage_backend/api/v1/routes/tasks.py 68
Traceback (most recent call last):
  File "/home/frank/eazy_manage_backend/api/v1/routes/tasks.py", line 57, in get_tasks
    serialized_data = [
  File "/home/frank/eazy_manage_backend/api/v1/routes/tasks.py", line 61, in <listcomp>
    'first_name': task[2].to_dict().first_name,
AttributeError: 'dict' object has no attribute 'first_name'
2024-10-24 12:17:06,077 INFO Logging setup successfully: /home/frank/eazy_manage_backend/./app.py 43
2024-10-24 12:17:06,078 INFO DataBase set up successfully: /home/frank/eazy_manage_backend/./app.py 55
2024-10-24 12:17:06,128 INFO database set up successfully: /home/frank/eazy_manage_backend/./app.py 60
2024-10-24 12:17:06,130 INFO Migrations set up successfully: /home/frank/eazy_manage_backend/./app.py 68
2024-10-24 12:17:06,136 INFO Blueprints registered successfully: /home/frank/eazy_manage_backend/./app.py 76
2024-10-24 12:17:06,136 INFO Mail service setup successfully: /home/frank/eazy_manage_backend/./app.py 89
2024-10-24 12:17:06,137 INFO App created successfully: /home/frank/eazy_manage_backend/./app.py 99
2024-10-24 12:19:09,745 INFO Logging setup successfully: /home/frank/eazy_manage_backend/./app.py 43
2024-10-24 12:19:09,745 INFO DataBase set up successfully: /home/frank/eazy_manage_backend/./app.py 55
2024-10-24 12:19:09,803 INFO database set up successfully: /home/frank/eazy_manage_backend/./app.py 60
2024-10-24 12:19:09,806 INFO Migrations set up successfully: /home/frank/eazy_manage_backend/./app.py 68
2024-10-24 12:19:09,811 INFO Blueprints registered successfully: /home/frank/eazy_manage_backend/./app.py 76
2024-10-24 12:19:09,812 INFO Mail service setup successfully: /home/frank/eazy_manage_backend/./app.py 89
2024-10-24 12:19:09,812 INFO App created successfully: /home/frank/eazy_manage_backend/./app.py 99
2024-10-24 12:19:14,350 WARNING Error fetching tasks: /home/frank/eazy_manage_backend/api/v1/routes/tasks.py 68
Traceback (most recent call last):
  File "/home/frank/eazy_manage_backend/api/v1/routes/tasks.py", line 57, in get_tasks
    serialized_data = [
  File "/home/frank/eazy_manage_backend/api/v1/routes/tasks.py", line 61, in <listcomp>
    'first_name': task[2].first_name.to_dict(),
AttributeError: 'str' object has no attribute 'to_dict'
2024-10-24 12:19:15,865 WARNING Error fetching tasks: /home/frank/eazy_manage_backend/api/v1/routes/tasks.py 68
Traceback (most recent call last):
  File "/home/frank/eazy_manage_backend/api/v1/routes/tasks.py", line 57, in get_tasks
    serialized_data = [
  File "/home/frank/eazy_manage_backend/api/v1/routes/tasks.py", line 61, in <listcomp>
    'first_name': task[2].first_name.to_dict(),
AttributeError: 'str' object has no attribute 'to_dict'
2024-10-24 12:20:03,730 INFO Logging setup successfully: /home/frank/eazy_manage_backend/./app.py 43
2024-10-24 12:20:03,730 INFO DataBase set up successfully: /home/frank/eazy_manage_backend/./app.py 55
2024-10-24 12:20:03,826 INFO database set up successfully: /home/frank/eazy_manage_backend/./app.py 60
2024-10-24 12:20:03,830 INFO Migrations set up successfully: /home/frank/eazy_manage_backend/./app.py 68
2024-10-24 12:20:03,836 INFO Blueprints registered successfully: /home/frank/eazy_manage_backend/./app.py 76
2024-10-24 12:20:03,837 INFO Mail service setup successfully: /home/frank/eazy_manage_backend/./app.py 89
2024-10-24 12:20:03,837 INFO App created successfully: /home/frank/eazy_manage_backend/./app.py 99
2024-10-24 12:21:02,893 INFO Logging setup successfully: /home/frank/eazy_manage_backend/./app.py 43
2024-10-24 12:21:02,894 INFO DataBase set up successfully: /home/frank/eazy_manage_backend/./app.py 55
2024-10-24 12:21:03,194 INFO database set up successfully: /home/frank/eazy_manage_backend/./app.py 60
2024-10-24 12:21:03,208 INFO Migrations set up successfully: /home/frank/eazy_manage_backend/./app.py 68
2024-10-24 12:21:03,244 INFO Blueprints registered successfully: /home/frank/eazy_manage_backend/./app.py 76
2024-10-24 12:21:03,245 INFO Mail service setup successfully: /home/frank/eazy_manage_backend/./app.py 89
2024-10-24 12:21:03,245 INFO App created successfully: /home/frank/eazy_manage_backend/./app.py 99
2024-10-24 12:21:52,007 INFO Logging setup successfully: /home/frank/eazy_manage_backend/./app.py 43
2024-10-24 12:21:52,008 INFO DataBase set up successfully: /home/frank/eazy_manage_backend/./app.py 55
2024-10-24 12:21:52,292 INFO database set up successfully: /home/frank/eazy_manage_backend/./app.py 60
2024-10-24 12:21:52,298 INFO Migrations set up successfully: /home/frank/eazy_manage_backend/./app.py 68
2024-10-24 12:21:52,314 INFO Blueprints registered successfully: /home/frank/eazy_manage_backend/./app.py 76
2024-10-24 12:21:52,314 INFO Mail service setup successfully: /home/frank/eazy_manage_backend/./app.py 89
2024-10-24 12:21:52,315 INFO App created successfully: /home/frank/eazy_manage_backend/./app.py 99
2024-10-24 12:26:29,126 INFO Logging setup successfully: /home/frank/eazy_manage_backend/./app.py 43
2024-10-24 12:26:29,127 INFO DataBase set up successfully: /home/frank/eazy_manage_backend/./app.py 55
2024-10-24 12:26:29,517 INFO database set up successfully: /home/frank/eazy_manage_backend/./app.py 60
2024-10-24 12:26:29,526 INFO Migrations set up successfully: /home/frank/eazy_manage_backend/./app.py 68
2024-10-24 12:26:29,551 INFO Blueprints registered successfully: /home/frank/eazy_manage_backend/./app.py 76
2024-10-24 12:26:29,552 INFO Mail service setup successfully: /home/frank/eazy_manage_backend/./app.py 89
2024-10-24 12:26:29,552 INFO App created successfully: /home/frank/eazy_manage_backend/./app.py 99
